packaging_automation/templates/docker/postgres-13/postgres-13.tmpl.dockerfile

```diff
@@ -1,7 +1,6 @@
-
 # This file is auto generated from it's template,
-# see citusdata/tools/packaging_automation/templates/docker/postgres-14/postgres-14.tmpl.dockerfile.
-FROM postgres:14rc1
+# see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
```

Comment:
PG 13 is not the latest major. I think there is a mistake in this comment here.

Reply:
+1, I'm also wondering why.

Reply:
Fixed the comment. Previously pg13 template was the latest so the comments remained as it was before.